Fixed v8::IdleNotification for nodejs 0.12

marcominetti opened the pull request.

deepak1556 committed:
update using nan and v0.11 support added

deepak1556 committed:
fix unreferenced handlescope close

deepak1556 committed:
Update README.md

Commit:
Fixed v8::IdleNotification for nodejs 0.12

Commit:
Fixed v8::IdleNotification for nodejs 0.12

Commit:
Update package.json

Commit:
Fixed v8::IdleNotification for nodejs 0.12

Using NAN.

FGRibreau commented:
:+1:

dgilperez commented:
:+1:

Globegitter commented:
:+1:

jwigley commented:
:+1:

andrewjmead commented:
:+1:

marcominetti commented:
Works on iojs too. ;) :+1: 

The idle_time_in_ms parameter for IdleNotification() is hardcoded and equal to 500ms. It is sufficient for me to make the gc run.

gierschv commented:
:+1:

jacoscaz commented:
Tested w/ node.js 0.12.0 and latest io.js, works like a charm.

lbeschastny commented:
:+1:

L2G commented:
Works for me, too. Running Node 0.12.0 on Mac OS X 10.9.5.

Commit:
Updated package.json file.

TooTallNate commented:
:+1:

matthewmueller commented:
:+1:

kieran-ohara commented:
+1

marcominetti commented:
available on npm as memwatch-next... waiting for pr merge... ;)

francisdb commented:
@marcominetti thanks :+1:

markusdap committed:
Link to the right git repository

Commit:
Merge pull request #1 from Globegitter/patch-1

Link to the right git repository

Commit:
Updated package.json file.

francisdb commented:
@marcominetti is this compatible with both 0.12 and 0.10?

francisdb commented:
@marcominetti can't do npm update when the path contains folders with spaces. This must have been introduced by your patch sice it works for memwatch 0.2.2

```
gyp ERR! stack Error: `make` failed with exit code: 2
```

marcominetti commented:
Fixing right now... wait some minutes... ;)

Commit:
Fixed nodegyp broken by path with spaces.

Commit:
Updated package.json file.

marcominetti commented:
@francisdb can you try now (version 0.2.5 in npm)? I've not tested on Windows...

francisdb commented:
@marcominetti works now here on linux / mac

jguepin committed:
Update module name in README

Commit:
Merge pull request #2 from chmod0/master

Update module name in README

Commit:
Update package.json

Commit:
Added iojs to Travis CI

Commit:
Update package.json

some-say commented:
:+1:

Zertz committed:
fix(bindings): support for npm@3 flatness

aleclarson commented:
:+1:

doron2402 committed:
fix readme

Commit:
Merge pull request #4 from doron2402/master

fix readme

Commit:
Merge pull request #3 from Zertz/patch-1

fix(bindings): support for npm@3 flatness

Commit:
Bump version 0.2.7

trshafer committed:
include iojs-v3 in travis ci build

Commit:
Upgraded NaN to 2.x.

Supporting now iojs 3.x branch.

Commit:
Fixed support for iojs/nodejs version < 3.x with Nan 2.x.

Commit:
Bump version 0.2.8

Commit:
Merge pull request #5 from amplii/build-iojsv3

include iojs-v3 in travis ci build

Commit:
Reworked gc() implementation.

Now uses both IdleNotification and LowMemoryNotification to ensure
'stats' event is triggered. The gc function now accepts custom deadline
in milliseconds for IdleNotification. In iojs/nodejs version 3.x the
full collection could not indeed occur.

Commit:
Bump version 0.2.9

Commit:
Merge branch 'master' of https://github.com/marcominetti/node-memwatch

toddbluhm committed:
Make compatible with Alpine linux by using cmath instead of math.h

Commit:
Merge pull request #6 from toddbluhm/master

Use std::isnan and std::isinf when on linux.

Commit:
Bump version 0.2.10

akras14 committed:
Update README.md

Minor, but JSON for heapdiff was missing one bracket.

Commit:
Merge pull request #7 from akras14/patch-1

Update README.md

Zertz committed:
chore(ci): add Node 4, 5 & 6

Commit:
Merge pull request #16 from Zertz/patch-1

chore(ci): add Node 4, 5 & 6

Commit:
Updated dependencies, travis targets and bump version 0.3.0.

Commit:
Fix travis ci build for nodejs 4+.

Commit:
Fix travis ci build for nodejs 4+.

varatep committed:
update license page

license page is previously incorrect

eklem committed:
aver --> after ?

Just a little typo?

Commit:
Merge pull request #29 from eklem/patch-1

aver --> after ?

Commit:
Merge pull request #21 from varatep/master

update license page

marcominetti committed:
Update README.md

jorisw committed:
Update README.md

Refer diretctly to @airbnb/node-memwatch

marcominetti committed:
Merge pull request #53 from jorisw/patch-1

Update README.md
